### PR TITLE
[Overview] Remove unwanted logo

### PR DESCRIPTION
**Pull request by iulianatodoran**
Removed unwanted logo, here is sample of the modified page 
image

### CODE REVIEW

**Comment by iulianatodoran** [comment]
@alttester/developers

File: Assets/AltTester/Runtime/AltDriver/AltDriver.cs [hunk 1]
```diff
@@ -52,7 +52,7 @@ public AltDriver(string host = "127.0.0.1", int port = 13000, string appName = "
             lock (driverLock)
             {
 #if UNITY_EDITOR || ALTTESTER
-            var defaultLevels = new Dictionary<AltLogger, AltLogLevel> { { AltLogger.File, AltLogLevel.Debug }, { AltLogger.Unity, AltLogLevel.Debug } };
+                var defaultLevels = new Dictionary<AltLogger, AltLogLevel> { { AltLogger.File, AltLogLevel.Debug }, { AltLogger.Unity, AltLogLevel.Debug } };
```
Comment (gomboskriszta15) [inline Assets/AltTester/Runtime/AltDriver/AltDriver.cs:55]: This file seems to have only the indentation modified. Could you revert this?

File: Assets/AltTester/Runtime/Input.cs [hunk 2]
```diff
@@ -559,13 +560,13 @@ public static void MoveTouch(int fingerId, Vector3 screenPosition)
     public static IEnumerator EndTouch(int fingerId)
     {
 #if UNITY_EDITOR
-            if (Application.isBatchMode)
-            {
-                yield return null;
-            }
-            else
+        if (Application.isBatchMode)
```
Comment (gomboskriszta15) [inline Assets/AltTester/Runtime/Input.cs:563]: Same here

File: Docs/source/pages/commands.md [hunk 3]
```diff
@@ -2713,15 +2713,15 @@ Removes key and its corresponding value from PlayerPrefs.
...
-    Test Delete Player Pref Key
-        Delete Player Pref
-        Set Player Pref Key           test                  1       String
-        ${actual_value}=              Get Player Pref Key   test    String
-        Should Be Equal As Strings    ${actual_value}       1
-        Delete Player Pref Key        test
-
-        Run Keyword And Expect Error    NotFoundException: PlayerPrefs key test not found    
-        ...    Get Player Pref Key      test    String
+        Test Delete Player Pref Key
```
Comment (gomboskriszta15) [inline Docs/source/pages/commands.md:2716]: Same here

**Review by RobertPoienar — APPROVED** [review]

**Review by andreion1ca — APPROVED** [review]

**Pull request merged**